youtube_dl_gui/locale/zh_CN/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,442 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Raymond Laud <htd0324@gmail.com>, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.3.8\n"
+"POT-Creation-Date: 2016-01-19 11:31 UT+8\n"
```

Comment:
You should change this to the original creation date which is `2015-07-02 15:31+EEST\n`.